egui/src/containers/window.rs

```diff
@@ -399,9 +405,11 @@ impl<'open> Window<'open> {
             content_inner
         };
 
-        area.state_mut().pos = ctx
-            .constrain_window_rect_to_area(area.state().rect(), area.drag_bounds())
-            .min;
+        if !fixed_pos {
+            area.state_mut().pos = ctx
+                .constrain_window_rect_to_area(area.state().rect(), area.drag_bounds())
+                .min;
```

Comment:
Perhaps we should let `area.end` do this instead by just re-setting `area.moveable` to the original value before calling `area.end` ? (area.end already constrains the area iff it is moveable)